enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/core/consensus/RaftMessages.java

```diff
@@ -59,12 +79,16 @@
         NEW_BATCH_REQUEST,
 
         PRUNE_REQUEST,
+
+        PRE_VOTE_REQUEST,
+        PRE_VOTE_RESPONSE,
     }
 
     interface RaftMessage extends Message
     {
         MemberId from();
         Type type();
+        <T, E extends Exception> T visit( Visitor<T, E> visitor ) throws E;
```

Comment:
I usually call this "dispatch", as in dispatch message onto correct handler.

Reply:
:+1: